extensions/sql-migration/src/constants/strings.ts

```diff
@@ -1427,6 +1427,8 @@ export const MIGRATION_SERVICE_DESCRIPTION = localize('sql.migration.select.serv
 
 // Desktop tabs
 export const DESKTOP_MIGRATION_BUTTON_LABEL = localize('sql.migration.tab.button.migration.label', 'New migration');
+export const DESKTOP_IMPORT_MIGRATION_BUTTON_LABEL = localize('sql.migration.tab.button.migration.label', 'Import migration');
```

Comment:
the "ID" (first parameter to the localize function) needs to be unique for every string